docs/sources/http_api/dashboard.md

```diff
@@ -541,3 +541,97 @@ Content-Length: 97
   "status": "multiple-slugs-exists"
 }
 ```
+
+## Trim input dashboard
+
+`POST /api/dashboards/trim`
+
+Will remove the default values from input dashboard JSON.
+
+**Example Request for trimming dashboard JSON**:
+
+```http
+POST /api/dashboards/uid/trim HTTP/1.1
```

Comment:
Uid shouldn't be included here right?

Reply:
no, we don't read from database, just based on the input, uid would not be needed